app/react/components/form-components/InputList/InputList.tsx

```diff
@@ -99,68 +91,86 @@ export function InputList<T = DefaultType>({
+          {value.map((item, index) => {
+            const key = itemKeyGetter(item, index);
+            const error =
+              typeof errors === 'object' ? errors[index] : undefined;
+
+            return (
+              <div
+                key={key}
+                className={clsx(
+                  styles.itemLine,
+                  { [styles.hasError]: !!error },
+                  'vertical-center'
+                )}
+              >
+                {Item ? (
+                  <Item
+                    item={item}
+                    onChange={(value: T) => handleChangeItem(key, value)}
+                    error={error}
+                    disabled={disabled}
+                    readOnly={readOnly}
+                  />
+                ) : (
+                  renderItem(
+                    item,
+                    (value: T) => handleChangeItem(key, value),
+                    error
+                  )
+                )}
+                <div className="items-start">
+                  {!readOnly && movable && (
+                    <>
+                      <Button
+                        size="medium"
+                        disabled={disabled || index === 0}
+                        onClick={() => handleMoveUp(index)}
+                        className="vertical-center btn-only-icon"
+                        icon={ArrowUp}
+                      />
+                      <Button
+                        size="medium"
+                        type="button"
+                        disabled={disabled || index === value.length - 1}
+                        onClick={() => handleMoveDown(index)}
+                        className="vertical-center btn-only-icon"
+                        icon={ArrowDown}
+                      />
+                    </>
+                  )}
+                  {!readOnly && (
                     <Button
+                      color="dangerlight"
                       size="medium"
-                      type="button"
-                      disabled={disabled || index === value.length - 1}
-                      onClick={() => handleMoveDown(index)}
+                      onClick={() => handleRemoveItem(key, item)}
                       className="vertical-center btn-only-icon"
-                      icon={ArrowDown}
+                      icon={Trash2}
                     />
-                  </>
-                )}
-                {!readOnly && (
-                  <Button
-                    color="dangerlight"
-                    size="medium"
-                    onClick={() => handleRemoveItem(key, item)}
-                    className="vertical-center btn-only-icon"
-                    icon={Trash2}
-                  />
-                )}
+                  )}
+                </div>
               </div>
-            </div>
-          );
-        })}
+            );
+          })}
+        </div>
+      )}
+      <div className="col-sm-12 mt-5">
```

Comment:
was this UX change discussed with PO? QA should test other views which use this component

Reply:
Yes, required by james to move the add button to the bottom of the list - I'll mention to testers the other places that use this component

Reply:
Thanks for mentioning this!

Reply:
https://portainer.atlassian.net/browse/EE-4918?focusedCommentId=23536